docs/community/release-process.rst

```diff
@@ -0,0 +1,51 @@
+Release Process and Rules
+=========================
+
+.. versionadded:: v2.6.2
+
+Starting with the version to be released after ``v2.6.2``, the following rules 
+will govern and describe how the Requests core team produces a new release.
+
+Major Releases
+--------------
+
+A major release will include breaking changes. When it is versioned, it will 
+be versioned as ``vX.0.0``. For example, if the previous release was 
+``v10.2.9`` the next version will be ``v11.0.0``.
+
+Breaking changes are changes that break backwards compatibility with prior 
+versions. If the project were to change the ``text`` attribute on a 
+``Response`` object to a method, that would only happen in a Major release.
+
+Major releases may also include miscellaneous bug fixes and upgrades to 
+vendored packages.
```

Comment:
maybe a note about "we don't do this often" / maybe once a year or once every 2 years.